build.go

```diff
@@ -98,6 +104,25 @@ func Build(entryResolver EntryResolver, dependencyManager DependencyManager, clo
 		logs.Action("Completed in %s", duration.Round(time.Millisecond))
 		logs.Break()
 
+		logs.GeneratingSBOM(goLayer.Path)
+		var sbomContent sbom.SBOM
+		duration, err = clock.Measure(func() error {
+			sbomContent, err = sbomGenerator.GenerateFromDependency(dependency, context.WorkingDir)
```

Comment:
```suggestion
			sbomContent, err = sbomGenerator.GenerateFromDependency(dependency, goLayer.Path)
```

I think this should be the path to the layer that contains the dependency since that is what it will effectively mean in the SBOM this method generates.